keel-core/src/main/kotlin/com/netflix/spinnaker/keel/api/DeliveryConfig.kt

```diff
@@ -10,7 +10,8 @@ data class DeliveryConfig(
   val name: String,
   val application: String,
   val artifacts: Set<DeliveryArtifact> = emptySet(),
-  val environments: Set<Environment> = emptySet()
+  val environments: Set<Environment> = emptySet(),
+  val apiVersion: ApiVersion = ApiVersion("delivery.config.spinnaker.netflix.com", "v1")
```

Comment:
question - is this value should be hardcoded? what should happen if we'll change to v2?

Reply:
That's just the default value. What that means is that the existing configs won't have to update - they'll automatically get the first version. But when we change in the future we will have some way of indicating it (this field) and processing it differently (not built yet).

Reply:
What's the likely mechanism of supporting old versions in a backwards compatible way? Should the DeliveryConfig class name itself be versioned and of a generic type?

Reply:
Great question - I'm not sure yet. I was tentatively thinking that there would be something that would convert the old config into the new version, save it as the new version, and return that back to the submitter.